pkg/data/components/OTelReceiver.yaml

```
@@ -43,6 +43,7 @@ properties:
     default: ${HTP_COLLECTOR_POD_IP}
     advanced: true
   - name: GRPCPort
+    display: GRPC Port
```

```suggestion
    display: gRPC Port
```
It's always weird deciding how to handle title case for abbreviations styled with a leading lowercase letter, but I think it's appropriate here to use the lowercase g.